src/components/pagination/Pagination.tsx

```diff
@@ -0,0 +1,215 @@
+import React, { FC, Fragment, ReactElement, MouseEvent } from 'react';
+import cn from 'classnames';
+
+import { GET_ERROR_MESSAGE } from 'constants/errorConstants';
```

Comment:
❗️
Это не функция, гет в названии лишний
```suggestion
import { ERROR_MESSAGE } from 'constants/errorConstants';
```